src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,24 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
 // Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <>
+    <article
```

Comment:
You have one parent tag anyway, no need for fragment
```suggestion
  <article
```

Reply:
Done.